frontend/lib/norent/letter-content.tsx

```diff
@@ -104,34 +104,41 @@ const PaymentDate = componentizeHelper((props) =>
   friendlyUTCDate(props.paymentDate)
 );
 
+const LandlordAddress: React.FC<NorentLetterContentProps> = (props) => (
+  <dd>
+    <LandlordName {...props} />
+    <br />
+    {props.landlordAddress ? (
+      <BreaksBetweenLines lines={props.landlordAddress} />
+    ) : (
+      <>{props.landlordEmail}</>
+    )}
+  </dd>
+);
+
+const Address: React.FC<NorentLetterContentProps> = (props) => (
+  <dd>
+    <FullName {...props} />
+    <br />
+    {getStreetWithApt(props)}
+    <br />
+    {props.city}, {props.state} {props.zipCode}
+    <br />
+    {formatPhoneNumber(props.phoneNumber)}
+  </dd>
+);
+
 /**
  * The to/from address of the letter.
- *
- * Note that this isn't internationalized because we don't actually
- * show it to the user in their locale.
  */
 const LetterHeading: React.FC<NorentLetterContentProps> = (props) => (
   <dl className="jf-letter-heading">
-    <dt>To</dt>
-    <dd>
-      <LandlordName {...props} />
-      <br />
-      {props.landlordAddress ? (
-        <BreaksBetweenLines lines={props.landlordAddress} />
-      ) : (
-        <>{props.landlordEmail}</>
-      )}
-    </dd>
-    <dt>From</dt>
-    <dd>
-      <FullName {...props} />
-      <br />
-      {getStreetWithApt(props)}
-      <br />
-      {props.city}, {props.state} {props.zipCode}
-      <br />
-      {formatPhoneNumber(props.phoneNumber)}
-    </dd>
+    <Trans description="heading of formal letter">
+      <dt>To</dt>
+      <LandlordAddress {...props} />
+      <dt>From</dt>
+      <Address {...props} />
+    </Trans>
```

Comment:
@sraby, I realized that we do actually store the entire localized letter on the _server-side_ for future presentation to the user, when they want to view their archived letters (this was added in #1473 if you remember reviewing that).  So I decided to follow your advice in https://github.com/JustFixNYC/tenants2/pull/1496#discussion_r431866272 and make the whole thing a `<Trans>` to provide better context, but I also factored out `<LandlordAddress>` and `<Address>` to make it less tag-soupy for translators.

Reply:
Nice! Yeah I think this is a perfect middle ground!